etc/profile-a-l/keepassxc.profile

```diff
@@ -62,8 +63,11 @@ include disable-xdg.inc
 #whitelist ${HOME}/.config/KeePassXCrc
 #include whitelist-common.inc
 
+mkdir ${RUNUSER}/app/org.keepassxc.KeePassXC
+whitelist ${RUNUSER}/app/org.keepassxc.KeePassXC
 whitelist /usr/share/keepassxc
 include whitelist-run-common.inc
+include whitelist-runuser-common.inc
```

Comment:
@rusty-snake [on Nov 5, 2022](https://github.com/netblue30/firejail/pull/5452#pullrequestreview-1169396646):

> >     whitelist ${RUNUSER}/app/org.keepassxc.KeePassXC

> This will break X11, D-Bus, ...

Do you think the whitelisting is safe with whitelist-run-user-common.inc
included?